core/src/test/scala/org/apache/spark/scheduler/DAGSchedulerSuite.scala

```diff
@@ -3742,7 +3742,8 @@ class DAGSchedulerSuite extends SparkFunSuite with TempLocalSparkContext with Ti
     runEvent(StageCancelled(0, Option("Explicit cancel check")))
     scheduler.handleShuffleMergeFinalized(shuffleMapStageToCancel,
       shuffleMapStageToCancel.shuffleDep.shuffleMergeId)
-    assert(mapOutputTracker.getNumAvailableMergeResults(shuffleDep.shuffleId) == 0)
+    assert(mapOutputTracker.getNumAvailableMergeResults(shuffleDep.shuffleId) == 2)
+    assert(shuffleMapStageToCancel.shuffleDep.isShuffleMergeFinalizedMarked)
```

Comment:
For indeterminate stages, we expect it to be unregistered (for same shuffleMergeId), right ?

Reply:
Yes, this UT is for determinate stage. The other UT checks both determinate and indeterminate stages

Reply:
Got it, can you update the test name to indicate this is for determinate stage ? Same for the other test.

Reply:
Updated the test method names